presto-hive/src/test/java/com/facebook/presto/hive/metastore/MockHiveMetastoreClient.java

```diff
@@ -126,43 +119,41 @@ public Table get_table(String dbName, String tableName)
     }
 
     @Override
-    public List<String> get_partition_names_ps(String dbName, String tableName, List<String> partValues, short maxParts)
-            throws TException
+    public List<String> getPartitionNamesPS(String dbName, String tableName, List<String> partValues,
+                                            short maxParts)
```

Comment:
Parameters should be formatted either all on one line or all wrapped. In this case, leave on same line.